clusterloader2/testing/load/config.yaml

```diff
@@ -29,7 +29,7 @@
 {{$CUSTOM_API_CALL_THRESHOLDS := DefaultParam .CUSTOM_API_CALL_THRESHOLDS ""}}
 {{$ENABLE_VIOLATIONS_FOR_API_CALL_PROMETHEUS := DefaultParam .CL2_ENABLE_VIOLATIONS_FOR_API_CALL_PROMETHEUS false}}
 #Variables
-{{$namespaces := DivideInt .Nodes $NODES_PER_NAMESPACE}}
+{{$namespaces := MultiplyInt .Nodes $NODES_PER_NAMESPACE}}
```

Comment:
This isn't correct - number of namespaces is #node divided by nodes per namespace.

Reply:
for example: you have 4 nodes（Nodes）, and specify that each node has 3 namespaces（NODES_PER_NAMESPACE）, then, the namespaces numbers should be:  4x3 = 12 namespaces， am I right?

Reply:
That's not what "NODES_PER_NAMESPACE" is for.

What you're talking about would be "NAMESPACES_PER_NODE", which we don't really have.

Reply:
Got it , thanks a lot.